site/content/en/contributions/RELEASING.md

```diff
@@ -97,34 +97,56 @@ export GITHUB_REMOTE=origin
    1. Create the release notes. Reference previous [release notes][] for additional details. __Note:__  The release
       notes should be an accumulation of the release candidate release notes and any changes since the release
       candidate.
-   2. Create a release announcement. Refer to [PR #635] as an example release announcement.
-   3. Include the release in the compatibility matrix. Refer to [PR #1002] as an example.
-   4. Generate the versioned release docs:
+   1. Create a release announcement. Refer to [PR #635] as an example release announcement.
```

Comment:
nit: those numbers are not correct

Reply:
https://github.com/envoyproxy/gateway/pull/635
https://github.com/envoyproxy/gateway/pull/1002